src/pages/ConditionsList/index.tsx

```diff
@@ -53,8 +53,17 @@ const customStyles = {
 }
 
 export const ConditionsList = () => {
-  const { data, error, loading } = useQuery<Conditions>(ConditionsListQuery)
+  const [searchTerm, setSearchTerm] = React.useState('')
+  const { data, error, loading } = useQuery<Conditions>(
+    searchTerm ? ConditionsSearchQuery : ConditionsListQuery,
+    {
+      variables: { conditionId: searchTerm },
```

Comment:
I saw the design, I founded the only property that we need to use to search, is the conditionId (see placeholder). That said , I think the _searchTerm_ property must be called (maybe) _conditionIdToSearch_ or something more clear to understand that the variable is related to "conditionIds"
![Screenshot_20200820_165428](https://user-images.githubusercontent.com/1144028/90819381-4da21c00-e306-11ea-8733-b7a440995b9d.png)